test/ui/README.md

```
@@ -0,0 +1,186 @@
+
+## Architecture
+
+- **Playwright** for browser automation
+- **TypeScript** with path aliases (`@config`, `@pages`, `@services`, etc.)
+- **Page Object Model** pattern for UI interactions
+- **Fixtures** for dependency injection (`oc` CLI service, unique name generator, cleanup tracker)
+- **Shared code reuse** via npm package
+
+This directory maps to it as follows:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)